ivy-hydra.el

```diff
@@ -34,11 +34,11 @@
 (require 'hydra)
 
 (defun ivy--matcher-desc ()
-  "Return description of `ivy--regex-function'."
-  (if (eq ivy--regex-function
-          'ivy--regex-fuzzy)
-      "fuzzy"
-    "ivy"))
+  "Return description of `ivy--regex-function'. "
+  (let ((cell (assoc ivy--regex-function ivy--preferred-re-builders)))
+    (if cell
+        (cdr cell)
```

Comment:
Should this be allowed through as is, or truncated past 5 columns (see hydra hint below)? Is it worth mentioning the limit anywhere?